engine/runner/src/main/java/org/enso/runner/Main.java

```diff
@@ -785,12 +790,16 @@ private void handleRun(
    * @param enableIrCaches are the IR caches enabled
    */
   private void genDocs(
-      String projectPath, Level logLevel, boolean logMasking, boolean enableIrCaches) {
-    if (projectPath.isEmpty()) {
-      println("Path hasn't been provided.");
+      String docsFormat,
+      String projectPath,
+      Level logLevel,
+      boolean logMasking,
+      boolean enableIrCaches) {
+    if (projectPath == null || projectPath.isEmpty()) {
+      println("Specify path to a project with --in-project option");
```

Comment:
Should probably print to `stderr` not `stdout`.

Reply:
A lot of messages were printed instead of being sent to stderr. I clean that all (up to my best knowledge) in 0d63f0a58f